docs/src/main/asciidoc/feature-management.adoc

```diff
@@ -0,0 +1,276 @@
+== Feature Management
+
+Feature Management provide a way for Spring Boot applications dynamically access content. Feature Management has a verity of functions, such as Feature Flags that can enable or disable content, Feature Flags for targeting when content is shown, and Feature Gates for dynamically enabling endpoints.
```

Comment:
The second item should be feature filters - I would add that those are based on targeted audiences, and include that we also have timewindow filters for exposing a feature for a limited amount of time.

Reply:
Is this the paragraph that introduces the feature management concept to customers? Then I think we should also include the information of when to use them and explain them in more depth, you can borrow this form the [concept doc on feature management](https://docs.microsoft.com/en-us/azure/azure-app-configuration/concept-feature-management)

Reply:
Feature Filters aren't always based on targeted audiences, I want to keep this intro generic.

Reply:
that is fine, then maybe call them "customized feature flags"? It is strange to read the same word defined twice in two different ways.